webui/assets/unsuspend.html

```diff
@@ -1,35 +1,38 @@
 <!doctype html>
 
-{{ template "head.html" }}
+{{ template "_head.html" }}
```

Comment:
❓ What are the differences between `webui/assets/unsuspend.html` and `webui/assets/suspend.html`?

Reply:
for now they are indeed the same files